packages/explorer/explorer-runner.mjs

```diff
@@ -0,0 +1,110 @@
+import { watchFile } from "fs";
+import { readFile } from "fs/promises";
+import minimist from "minimist";
+import path from "path";
+import process from "process";
+import { fileURLToPath } from "url";
+import { spawn } from "child_process";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const argv = minimist(process.argv.slice(2));
+const port = argv.port || 13690;
+const chainId = argv.chainId || 31337;
+const env = argv.env || "production";
+const indexerDbPath = argv.indexerDbPath || "indexer.db";
+const worldsConfigPath = argv.worldsConfigPath || null;
+
+let worldAddress = argv.worldAddress || null;
+let explorerProcess;
+
+async function startExplorer() {
+  let command, args;
+
+  if (env === "production") {
+    command = "pnpm";
+    args = ["start"];
+  } else {
+    command = "pnpm";
+    args = ["dev"];
+  }
+
+  explorerProcess = spawn(command, args, {
+    cwd: __dirname,
+    stdio: "inherit",
+    env: {
+      ...process.env,
+      PORT: port,
+      INIT_PWD: process.cwd(),
+      WORLD_ADDRESS: worldAddress,
+      INDEXER_DB_PATH: indexerDbPath,
```

Comment:
rather than feeding in `INIT_PWD` maybe we just do a `path.join(process.cwd(), indexerDbPath)` here?